payments-core/src/main/java/com/stripe/android/payments/core/authentication/DefaultIntentAuthenticatorRegistry.kt

```diff
@@ -21,47 +18,23 @@ import kotlin.coroutines.CoroutineContext
  * Default registry to provide look ups for [IntentAuthenticator].
  * Should be only accessed through [DefaultIntentAuthenticatorRegistry.createInstance].
  */
-internal class DefaultIntentAuthenticatorRegistry @Inject internal constructor() :
-    IntentAuthenticatorRegistry {
-
-    // default authenticators always packaged with the SDK
-    // TODO(ccen) register a singleton instance of each authenticator with registration API
-    @Inject
```

Comment:
instead of directly injecting the actual `IntentAuthenticator` instances, now we injected a map to map `NextActionData` to `IntentAuthenticator`s